pyDive/pyDive.py

```diff
@@ -21,8 +21,6 @@
 globals().update(ndarray_mod.factories)
 globals().update(ndarray_mod.ufuncs)
 from .arrays.ndarray import ndarray  # noqa
-# often used functions for ndarrays
-from .arrays.ndarray_funcs import *
```

Comment:
F403 'from arrays.ndarray_funcs import *' used; unable to detect undefined names